conda/meta.yaml

```diff
@@ -0,0 +1,58 @@
+{% set data= load_setup_py_data() %}
+package:
+  name: ts-interlockmonitor
+  version: {{ data.get('version') }}
+
+source:
+  path: ../
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install --no-deps --ignore-installed .
+  script_env:
+    - PATH
+    - PYTHONPATH
+    - LD_LIBRARY_PATH
+    - OSPL_HOME
+    - PYTHON_BUILD_VERSION
+    - PYTHON_BUILD_LOCATION
+    - LSST_DDS_DOMAIN
+
+test:
+  requires:
+    - ts-conda-build
+    - ts-dds
+    - ts-idl {{ idl_version }}
+    - ts-salobj {{ salobj_version }}
+
+  source_files:
+    - python
+    - bin
+    - tests
+    - setup.cfg
+    - pyproject.toml
+  commands:
+      - pytest
+      - start-thermal-server --help
+
+requirements:
+  host:
+    - python {{ python }}
+    - pip
+    - setuptools_scm
+    - setuptools
+      
+  build:
+    - python {{ python }}
+    - setuptools_scm
+    - setuptools
+    - ts-conda-build =0.3
+  run:
+    - python {{ python }}
+    - setuptools
+    - setuptools_scm
+    - typer
```

Comment:
Technically typer is provided with the LSST conda environment or it is the Vera Rubin Conda environment.

Reply:
Please don't use it here without a discussion. In this case the API is simple so I suggest argparse. It works and it's in the standard library so we don't have to worry about versioning. If you hate argparse, then let's have a public discussion about T&S approved CLI libraries since I would like to know what the allowed and preferred options are.